pinot-core/src/main/java/com/linkedin/pinot/core/query/aggregation/function/PercentileEstAggregationFunction.java

```diff
@@ -15,27 +15,57 @@
  */
 package com.linkedin.pinot.core.query.aggregation.function;
 
+import com.clearspring.analytics.stream.quantile.TDigest;
 import com.linkedin.pinot.common.utils.DataSchema;
 import com.linkedin.pinot.core.common.BlockValSet;
 import com.linkedin.pinot.core.query.aggregation.AggregationResultHolder;
 import com.linkedin.pinot.core.query.aggregation.ObjectAggregationResultHolder;
-import com.linkedin.pinot.core.query.aggregation.function.customobject.QuantileDigest;
 import com.linkedin.pinot.core.query.aggregation.groupby.GroupByResultHolder;
 import com.linkedin.pinot.core.query.aggregation.groupby.ObjectGroupByResultHolder;
+import java.nio.ByteBuffer;
 import javax.annotation.Nonnull;
 
 
-public class PercentileEstAggregationFunction implements AggregationFunction<QuantileDigest, Long> {
-  public static final double DEFAULT_MAX_ERROR = 0.05;
+/**
+ * TDigest based Percentile aggregation function.
+ */
+public class PercentileEstAggregationFunction implements AggregationFunction<TDigest, Double> {
```

Comment:
Are you sure we want to replace the current one? There will be backward compatibility issue

Reply:
There are no usages of this, as far as we can tell.